src/core_atmosphere/mpas_atm_core.F

```diff
@@ -405,7 +405,9 @@ subroutine atm_mpas_init_block(dminfo, stream_manager, block, mesh, dt)
       real (kind=RKIND), dimension(:), pointer :: dvEdge, invDvEdge
       real (kind=RKIND), dimension(:), pointer :: dcEdge, invDcEdge
       real (kind=RKIND), dimension(:), pointer :: areaTriangle, invAreaTriangle
-      integer, pointer :: nCells, nEdges, nVertices, nVertLevels, nEdgesSolve
+      integer, pointer :: nCells_ptr, nEdges_ptr, nVertices_ptr, nVertLevels_ptr, nEdgesSolve
+      integer :: nCells, nEdges, nVertices, nVertLevels
+      integer :: k
```

Comment:
Seems cleaner to put `k` with the other loop counters above; do `integer :: iCell,iEdge,iVertex,k`.

Reply:
done